src/accessibility/describe.js

```diff
@@ -0,0 +1,259 @@
+/**
+ * @module Environment
+ * @submodule Environment
+ * @for p5
+ * @requires core
+ */
+
+import p5 from '../core/main';
+
+/**
+ * Creates a screen-reader accessible description for the canvas in the DOM.
```

Comment:
Thinking of a new coder learning about this function. Maybe the part about the DOM can go at the end? I think a new programmer could still use this function without knowing about the DOM and the acronym might be intimidating. I think it's important enough to mention here, but maybe it can go at the end, like it's supplemental info.